<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMapDialog</class>
 <widget class="QDialog" name="NewMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="mSceneGB">
         <property name="title">
          <string>Description</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QFormLayout" name="formLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="mMapNameLE"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Creator</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="mCreatorLE"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Max Players</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="mMaxPlayersSB">
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>16</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="mColorLbl">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Background Color</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="ClickableFrame" name="mColorBtn">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
                <property name="frameShape">
                 <enum>QFrame::Panel</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <property name="lineWidth">
                 <number>2</number>
                </property>
                <property name="midLineWidth">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Notes</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="mSceneNotesPTE"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="mLoadGridGB">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You have the option to create a base layer with a tile of your choice.. This is, a lot of times, a good idea because it helps you orient yourself when you are creating a new map. You can always erase portions you don't want.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="title">
          <string>Load base grid layer</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="formLayout_4">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Grid size</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSpinBox" name="mGridXSB"/>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>by</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="mGridYSB"/>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Height</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="mHeightSB"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Tile Name</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="mTileNameLE">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="mNewTileBtn">
                <property name="text">
                 <string>New</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="mChooseTileBtn">
                <property name="text">
                 <string>Choose</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="mTilePluginLE">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Plugin</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mLoadSkyboxGB">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Load a skybox as the background - if you already have some skyboxes that you have made you can use one of them here. A map can only use one skybox.. which makes sense. Just keep that in mind.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="title">
          <string>Load skybox</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="mSkyboxLE">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="mNewSkyboxBtn">
                <property name="text">
                 <string>New</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="mChooseSkyboxBtn">
                <property name="text">
                 <string>Choose</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="mSkyboxPluginLE">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Plugin</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Directional light</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="mLightNameLbl">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="mDirLightLE">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="mNewDirLightBtn">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>New</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="mChooseLightBtn">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Choose</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Create default</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="mCreateDefaultDirLightCB">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="mLightPluginLE">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="mLightPluginLbl">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Plugin</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Camera</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Create default</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="mCreateDefaultCamCB">
              <property name="layoutDirection">
               <enum>Qt::RightToLeft</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="mCamNameLbl">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="mCamLE">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="mNewCamBtn">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>New</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="mChooseCamBtn">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Choose</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="mCamPluginLE">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="mCamPluginLbl">
              <property name="text">
               <string>Plugin</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mCreateBtn">
       <property name="text">
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>label_6</zorder>
  <zorder></zorder>
  <zorder></zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableFrame</class>
   <extends>QFrame</extends>
   <header>clickableframe.h</header>
   <container>1</container>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mNewTileBtn</sender>
   <signal>pressed()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onNewTile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mNewSkyboxBtn</sender>
   <signal>pressed()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onNewSkybox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>479</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mChooseTileBtn</sender>
   <signal>pressed()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onChooseTile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>497</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mChooseSkyboxBtn</sender>
   <signal>pressed()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onChooseSkybox()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCreateBtn</sender>
   <signal>pressed()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onCreate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancelBtn</sender>
   <signal>pressed()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mNewDirLightBtn</sender>
   <signal>pressed()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onNewDirlight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mChooseLightBtn</sender>
   <signal>pressed()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onChooseDirlight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>580</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mNewCamBtn</sender>
   <signal>pressed()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onNewCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mChooseCamBtn</sender>
   <signal>pressed()</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onChooseCamera()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>454</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCreateDefaultCamCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onCameraCB(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>554</x>
     <y>405</y>
    </hint>
    <hint type="destinationlabel">
     <x>17</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCreateDefaultDirLightCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>NewMapDialog</receiver>
   <slot>onDirLightCB(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onNewTile()</slot>
  <slot>onChooseTile()</slot>
  <slot>onNewSkybox()</slot>
  <slot>onChooseSkybox()</slot>
  <slot>onCreate()</slot>
  <slot>onCancel()</slot>
  <slot>onNewDirlight()</slot>
  <slot>onChooseDirlight()</slot>
  <slot>onNewCamera()</slot>
  <slot>onChooseCamera()</slot>
  <slot>onDirLightCB(bool)</slot>
  <slot>onCameraCB(bool)</slot>
 </slots>
</ui>
